### PR TITLE
Update hyperlink to open in new tab

### PR DESCRIPTION
**Pull request by vinay-04**
# Description
Update target to `_blank`,  now the link will open in a new tab.

## Type of change

- [x] Chore (non-breaking change which adds functionality)

## Changes

```
+     target="_blank"
```
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: client/dashboard/app/routes/applications.status.$domain.tsx [hunk 1]
```diff
@@ -129,6 +129,7 @@ export default function Application() {
               <h5>
                 Not yet joined the super awesome SRMKZILLA Discord?{" "}
                 <a
+                  target="_blank"
```
[the remark this answers is not in the document]
Comment (vinay-04) [reply client/dashboard/app/routes/applications.status.$domain.tsx:132]: I'm unsure about the differences, but I guess `_blank` is the standardised value for "target"

[the remark this answers is not in the document]
Comment (vinay-04) [reply client/dashboard/app/routes/applications.status.$domain.tsx:132]: After researching online I came to the conclusion that if `target="_blank"` is specified in the anchor tag, and assuming multiple links are present in the file, then each link will open a new tab, whereas if `target="blank"` is specified, then each link with this argument will open one new tab for the first link clicked, and subsequent links will open their content in the same tab previously loaded.

**Comment by shawshankkumar** [comment]
Yes!
I think that's better here. Please update it and let me know.
Thanks

**Comment by vinay-04** [comment]
Just to confirm, I got to update 'blank' instead of '_blank' right?

**Comment by shawshankkumar** [comment]
> Just to confirm, I got to update 'blank' instead of '_blank' right?

Yes!

**Comment by shawshankkumar** [comment]
@vinay-04 once you are done maing the changes, please re-request the review. Thanks!

**Comment by shawshankkumar** [comment]
LGTM! 🚀 

@SaanviKumar13 merging this, please test it and let me know.

**Review by shawshankkumar — APPROVED** [review]

**Pull request merged**